common/dynamicconfig/constants.go

```diff
@@ -582,6 +582,30 @@ const (
 	// Default value: 100000
 	// Allowed filters: DomainName
 	FrontendGlobalDomainAsyncRPS
+	// FrontendGlobalRatelimiterDomainUserRPS is workflow domain rate limit per second for the whole Cadence cluster
+	// KeyName: frontend.globalRatelimiterDomainrps
+	// Value type: Int
+	// Default value: 1000
+	// Allowed filters: DomainName
+	FrontendGlobalRatelimiterDomainUserRPS
+	// FrontendGlobalRatelimiterDomainWorkerRPS is background-processing workflow domain rate limit per second for the whole Cadence cluster
```

Comment:
I would rephrase this slightly to include information that it's purpose is to throttle the worker poll/throttle/process loop. It's not immediately clear, reading this, whether this is referring to some internal rate-lmiit inside history or not (it's not)

Reply:
these have been dropped in favor of reusing the existing `FrontendGlobalDomainUserRPS` configs (all 4 the same, just drop `Ratelimiter` from the middle).

though I agree in principle, for the kinda-vague-ness of these configs.  I should try to clear up those older ones.

Reply:
updated the comments for these limits